app/controllers/api/v1/users/sessions_controller.rb

```diff
@@ -2,6 +2,10 @@ class Api::V1::Users::SessionsController < Api::V1::BaseController
   def create
     load_resource
     if @user
+
+      @user.regenerate_session_token!
+      @user.regenerate_refresh_token!
+
       render json: Api::V1::SessionBlueprint.render(@user), status: 201
     else
       render json: {message: "Wrong password or email"}, status: 401
```

Comment:
optional: let's update the error message to "Incorrect email or password."
similiar to how other apps do it.

![image](https://github.com/user-attachments/assets/bb23478b-097b-485c-a0c9-190e64564f07)